app/src/main/java/homer/common/TemperatureFactoryImpl.java

```diff
@@ -0,0 +1,22 @@
+package homer.common;
+
+public final class TemperatureFactoryImpl implements TemperatureFactory {
+    private static final double DELTA_KELVIN_CELSIUS = 273.15;
+    private static final double DELTA_FARENHEIT_KELVIN = 459.67;
```

Comment:
The same constants appear in KelvinTemperature, maybe they should be moved to the Temperature interface?

Reply:
ok, i can move them.